docs/howto/manage-web-app-charms/configure-web-app-charm.rst

```diff
@@ -85,8 +85,132 @@ The configuration can be set on the deployed charm using:
 
     juju config <app name> token=<token>
 
-Manage secrets for a 12-factor app charm
-----------------------------------------
+Add a custom action
+-------------------
+
+For certain commands or processes that you will run many times, you can set up
+custom Juju actions to expedite your workflow. Actions are handled in the Ops
+library, and Juju manages the action's workflow via tasks and operations. To
+implement a custom action, you need to declare the action in
+``charmcraft.yaml`` and define an event handler into the Ops framework of your
```

Comment:
> define... into

"into" wouldn't be my choice for a preposition. Should it be "for"? Or maybe "add... to"?